concepticondata/conceptlists/Speed-2017-485.tsv

```diff
@@ -0,0 +1,486 @@
+Speed-2017-485-74	74	citroengras	lemongrass	0.066666667	2.333333333	3.2	0.666666667	1.866666667	ruiken	0.385245902	3.666666667	3.166666667	4.666666667		
+Speed-2017-485-75	75	curry	curry	0.266666667	4.466666667	3.466666667	0.666666667	2.8	proeven	0.36	4.333333333	4.416666667	5.166666667		
+Speed-2017-485-76	76	damp	vapor	0.285714286	0	0.214285714	2.428571429	3.285714286	zien	0.528735632	3.333333333	3.75	3.583333333	1193	STEAM
+Speed-2017-485-77	77	damschijven	sheet piles	0.733333333	0	0	2.333333333	3.866666667	zien	0.557692308	3.583333333	3.166666667	3.75		
+Speed-2017-485-78	78	dekbed	comforter	0.533333333	0	0.466666667	4.2	3.266666667	voelen	0.496062992	2.416666667	2.833333333	5.5		
+Speed-2017-485-79	79	deken	dean	0.2	0	0.333333333	4.2	3.266666667	voelen	0.525	2.416666667	3	5.333333333		
+Speed-2017-485-80	80	den	den	0.2	0	2.6	1.133333333	3.8	zien	0.49137931	3.333333333	4.083333333	5.083333333	2949	LAIR
+Speed-2017-485-81	81	dennenappel	pine cone	0.2	0	1.8	2.333333333	3.866666667	zien	0.471544715	3.25	3.833333333	4.5		
+Speed-2017-485-82	82	deurbel	doorbell	4.066666667	0	0.333333333	1.533333333	1.866666667	horen	0.521367521	5.416666667	5.666666667	4.083333333		
+Speed-2017-485-83	83	deurklink	door handle	1.133333333	0	0	2.866666667	3.2	zien	0.444444444	3.333333333	3.333333333	3.416666667		
+Speed-2017-485-84	84	deurknop	doorknob	0.6	0	0	3.133333333	3.266666667	zien	0.466666667	3.583333333	3.666666667	3.916666667		
+Speed-2017-485-85	85	didgeridoo	didgeridoo	3.857142857	0.357142857	0.357142857	1.071428571	3	horen	0.404958678	3.75	4.5	3.666666667		
+Speed-2017-485-86	86	doek	cloth	0	0	0.333333333	2.533333333	3.866666667	zien	0.574257426	3.333333333	2.833333333	4.5	1616	CLOTH
+Speed-2017-485-87	87	dominostenen	dominoes	1.266666667	0	0	1.8	3.933333333	zien	0.561904762	3.5	3.416666667	4.083333333	2594	DOMINOES
+Speed-2017-485-88	88	donder	thunder	4.785714286	0	0.214285714	0.285714286	2.142857143	horen	0.644230769	4.666666667	6.25	3.916666667	1150	THUNDER
+Speed-2017-485-89	89	doolhof	maze	0.642857143	0	0.071428571	1.071428571	4.214285714	zien	0.702380952	5.5	4.083333333	4.166666667		
+Speed-2017-485-90	90	dragon	tarragon	0.166666667	2.916666667	2.833333333	0.583333333	2.25	proeven	0.314285714	4.5	4.75	3.416666667		
+Speed-2017-485-91	91	driehoek	triangle	0.133333333	0	0	1	4.333333333	zien	0.792682927	3.416666667	3.5	3.916666667		
+Speed-2017-485-92	92	droger	dryer	2.8	0	0.533333333	1	3.4	zien	0.439655172	4.416666667	4.5	4.5		
+Speed-2017-485-93	93	drop	drop	0	3.733333333	1.533333333	1.866666667	3.266666667	proeven	0.358974359	3.416666667	4.5	4.333333333	3748	DROP (OF A LIQUID)
+Speed-2017-485-94	94	druif	grape	0	4.533333333	1.933333333	1.733333333	3.333333333	proeven	0.393063584	4.166666667	3.25	5.083333333	920	GRAPE
+Speed-2017-485-95	95	duik	dive	1.6	0	0.133333333	3.066666667	3.6	zien	0.428571429	4.75	4.333333333	5.083333333		
```

Comment:
DIVE?

Reply:
The authors say that the list only includes nouns. That's why I did not map it.